contracts/libraries/TokenConservation.sol

```diff
@@ -10,10 +10,10 @@ library TokenConservation {
     using SignedSafeMath for int256;
 
     /** @dev initialize the token conservation data structure
-      * @param tokenIdsForPrice sorted list of tokenIds for which token conservation should be checked
+      * @param tokenNumber is the number of tokens the conservation should be checked for
       */
-    function init(uint16[] memory tokenIdsForPrice) internal pure returns (int256[] memory) {
-        return new int256[](tokenIdsForPrice.length + 1);
+    function init(uint256 tokenNumber) internal pure returns (int256[] memory) {
```

Comment:
`tokenNumber` implies its the number of a single token, I think `tokenCount` is more appropriate here.